ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -410,6 +418,7 @@ class MSFNotificationStateImpl: NSObject, ControlConfiguration, MSFNotificationS
 
     @objc init(style: MSFNotificationStyle) {
         self.style = style
+        self.showDefaultDismissActionButton = style.isToast
```

Comment:
is this needed if you are double checking in line 455?

Reply:
This one doesn't call the convenience init, so it isn't double checking on 455.

Reply:
hmmm everything should evenually funnel to one init? so we don't have duplicated stuff?